src/toolbar/HelperWidgets.h

```diff
@@ -0,0 +1,199 @@
+#pragma once
+
+inline const char* GetSpecialKeyName(UINT aVKCode)
```

Comment:
Why is all of this inlined?

Reply:
moved to cpp, also moved GetSpecialKeyName into VKBindings, as i think it is better place for this than was here